### DISCUSSION TITLE
Rule to prevent overhead space before statement

### DISCUSSION THREAD

**Discussion by dmitryuk** [PHP-CS-Fixer/PHP-CS-Fixer#8128] [Q&A] [answered]
Example code
```

$exampleCode           = 1;

```

expected

```

$exampleCode = 1;

```

**Comment by Wirone** [comment, marked as answer]
Please use [`binary_operator_spaces`](https://cs.symfony.com/doc/rules/operator/binary_operator_spaces.html) (and review [rules](https://cs.symfony.com/doc/rules/index.html) in general before asking, thanks for understanding 🙏).
